Without at least -O1 optimizations, many security checks just do not work

so default to at least O1

--- gcc-6.2.0/gcc/opts.c~	2016-03-17 15:11:35.000000000 +0000
+++ gcc-6.2.0/gcc/opts.c	2016-11-19 19:01:38.701907744 +0000
@@ -558,6 +558,8 @@
   int opt2;
   bool openacc_mode = false;
 
+  opts->x_optimize = 1;
+
   /* Scan to see what optimization level has been specified.  That will
      determine the default value of many flags.  */
   for (i = 1; i < decoded_options_count; i++)
